MAUI/CheckBox/Visual-Customization.md

```diff
@@ -262,4 +262,41 @@ The [`ControlSize`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Buttons.
     this.Content = stackLayout;
 
 {% endhighlight %}
-{% endtabs %}
\ No newline at end of file
+{% endtabs %}
+## EnabledAnimation
+
+The `EnabledAnimation` property is used to enable or disable the animation for the [`CheckBox`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Buttons.SfCheckBox.html) control. 
+
+{% tabs %}
+{% highlight xaml %}
+
+ <StackLayout>
+     <syncfusion:SfCheckBox Text="CheckBox EnabledAnimation True" EnabledAnimation="True"/>
+     <syncfusion:SfCheckBox Text="CheckBox DisabledAnimation False" EnabledAnimation="False"/>
```

Comment:
Add any one check box

Reply:
Removed the outputs and change the checkbox text